deployment/terraform/environments/eu-central-1-prod.tfvars

```diff
@@ -0,0 +1,9 @@
+aws_region                = "eu-central-1"
+ENVIRONMENT               = "prod"
+vpc_cidr_block            = "10.0.0.0/16"
+subnet_cidr_block         = "10.0.1.0/24"
+private_subnet_cidr_block = "10.0.2.0/24"
+ami_number                = "ami-0abcd1234efgh5678"
+instance_name             = "web_app_prod"
```

Comment:
for  now it's ok, but name should not contain env, you can concatenate name  from $instance_name-$env in other tf files

Reply:
I hope the refactor is correct